build/Packages.targets

```diff
@@ -13,6 +13,7 @@
       https://dotnet.myget.org/F/project-system-interop/api/v3/index.json;
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://vside.myget.org/F/vssdk/api/v3/index.json;
+      C:\Users\tomescht\Documents\LocalPackageSource
```

Comment:
This will be removed before the work is merged in, once suitable official packages are available for consumption.